### PR TITLE
Fix for download:ledger

### LINKED ISSUE #38: ledger:download broken
> evently ledger:download -f test.ndjson -t $EVENTlY_TOKEN                                                                                                                         14:22:48
no ledger file at test.ndjson
Downloading ledger fully.
    TypeError: Invalid state: ReadableStream is locked
    Code: ERR_INVALID_STATE

Is this a fix that has to come from Ketting?

### PR DESCRIPTION
**Pull request by mattbishop**
Fixes #38

### CODE REVIEW

File: src/commands/ledger/download.ts [hunk 1]
```diff
@@ -26,7 +26,7 @@ Validated 13,438 ledger events.
 
     initClient(flags.token)
 
-    const count = await downloadAndValidateLedger(flags.file)
```
Comment (evert) [inline src/commands/ledger/download.ts:29]: I'm thinking this caused the test to fail. Ketting by default already adds the token, so you don't need to pass it here.

**Comment by mattbishop** [comment]
Sigh I forgot about running the tests.

I'll have to mock both ketting and fetch to get it to work.

**Comment by evert** [comment]
Happy to write tests for this!

**Comment by evert** [comment]
Ignore my last commit, I wasn't fully sure what you were trying to do so I partially reverted it and just reintroduced the bug.

However, I have found the real issue. You were right, Ketting reads the response before we get a chance to open the stream. The specific feature that causes this is the server responding with `Content-Location`. When this header appears, Ketting will read the entire stream and put it in cache.

We clearly don't want this for large responses, so I have to find a way to disable this for specific requests. It's not ideal either way I realize now, so it's just an interaction I unfortunately haven't thought of.

**Comment by evert** [comment]
https://github.com/badgateway/ketting/issues/474

**Pull request closed without merging**